src/hotspot/cpu/riscv/c1_Runtime1_riscv.cpp

```diff
@@ -498,6 +498,15 @@ void Runtime1::generate_unwind_exception(StubAssembler *sasm) {
   // other registers used in this stub
   const Register handler_addr = x11;
 
+  if (AbortVMOnException) {
+    __ mv(x15, x10);
+    __ enter();
+    save_live_registers(sasm);
+    __ call_VM_leaf(CAST_FROM_FN_PTR(address, check_abort_on_vm_exception), x15);
+    restore_live_registers(sasm);
+    __ leave();
+  }
+
```

Comment:
Hi, the newly-added test case passed on linux-riscv64 with this fix.
But seems that this could still be improved? Something like:
```
diff --git a/src/hotspot/cpu/riscv/c1_Runtime1_riscv.cpp b/src/hotspot/cpu/riscv/c1_Runtime1_riscv.cpp
index 24b97875910..ea086d46bda 100644
--- a/src/hotspot/cpu/riscv/c1_Runtime1_riscv.cpp
+++ b/src/hotspot/cpu/riscv/c1_Runtime1_riscv.cpp
@@ -499,10 +499,9 @@ void Runtime1::generate_unwind_exception(StubAssembler *sasm) {
   const Register handler_addr = x11;

   if (AbortVMOnException) {
-    __ mv(x15, x10);
     __ enter();
     save_live_registers(sasm);
-    __ call_VM_leaf(CAST_FROM_FN_PTR(address, check_abort_on_vm_exception), x15);
+    __ call_VM_leaf(CAST_FROM_FN_PTR(address, check_abort_on_vm_exception), x10);
     restore_live_registers(sasm);
     __ leave();
   }
```

Reply:
@RealFYang you're right: no need to copy the argument to another register. Fixed it. Thank you!